lib/internal/process/task_queues.js

```diff
@@ -172,7 +171,7 @@ function queueMicrotask(callback) {
   const asyncResource = createMicrotaskResource();
   asyncResource.callback = callback;
 
-  enqueueMicrotask(FunctionPrototype.bind(runMicrotask, asyncResource));
+  enqueueMicrotask(FunctionPrototypeBind(runMicrotask, asyncResource));
```

Comment:
Same question here.
Are we actually doing something wrong with the primordials so that V8 doesn't know that the properties never change?

Reply:
Not sure. @mcollina flagged this for me and I've noticed it's definitely slower. He might have more insights — not sure if he talked to the V8 team about it at all.

Reply:
There is an ongoing effort to optimize frozen object performance, see https://bugs.chromium.org/p/v8/issues/detail?id=6831 we may revisit this when the patches upstream land here.

Reply:
> Are we actually doing something wrong with the primordials so that V8 doesn't know that the properties never change?

We are. Essentially accessing them is slower than accessing normal objects. We should look on _not_ reading properties in hot code path, but maybe only once when the module is loaded.